packages/block-library/src/code/block.json

```diff
@@ -14,6 +14,7 @@
 	},
 	"supports": {
 		"anchor": true,
+		"__experimentalSelector": ".wp-block-code > code",
```

Comment:
The issue with this is that the styles from global styles applies to the inner "code" but not the styles from the "fontSize" block support and any block support we add here in the future.

Reply:
Good point, should we revert this PR until we figure out an alternative?